syntax = "proto3";

service FileTransfer {
    rpc SendFile(FileRequest) returns (FileResponse);
}

message FileRequest {
    string file_name = 1;
    bytes file_content = 2;
}

message FileResponse {
    string message = 1;
}

